### ISSUE TITLE
[Core][fd-menu] ACC-270.16 (Level A)items in fd-menu should not be triggered when key down

### ISSUE THREAD

**Issue by Meggie886** [SAP/fundamental-ngx#14094]
### Describe your proposal

Items in fd-menu should be triggered when key up

### Proposed Solution

Items in fd-menu should be triggered when key up

<img width="1848" height="666" alt="Image" src="https://github.com/user-attachments/assets/160ad8a0-64fc-4018-9a2e-e6ffdca45203" />
Set click and keyUp event the same time does not help. keyUp event only works when I remove the click event. We should support both or when click event is set, should be trigger when keyup rather than keydown.

https://stackblitz.com/edit/qzeeq1ew?file=src%2Fapp%2Fmenu-placement-example.component.ts

### Priority

None

### Declaration

- [x] I'm not disclosing any internal or sensitive information.

**Comment by MariaIDineva** [comment]
Hi, @Meggie886 !
Thank you for reporting this issue. Can you please check out the fix we are proposing here - https://deploy-preview-14115--fundamental-ngx.netlify.app/#/core/menu#placement, under this temporary example named `Keyboard Activation (keyup fix)` that I've added in order to test the issue? Please, let us now if we now have the desired behaviour or if there is anything else that we have to change.
Thanks!
